src/dev-spaces/azext_dev_spaces/azext_metadata.json

```diff
@@ -1,3 +1,3 @@
 {
-    "azext.minCliCoreVersion": "2.0.63"
```

Comment:
If you set `"azext.minCliCoreVersion": "2.1.0"`, the extension will be only available when azure-cli >= 2.1.0.
Are you sure about it?.

Reply:
Yes. I have validated the changes with the 2.1.0 version.